code/ui/blocks/src/components/Story.tsx

```diff
@@ -44,7 +45,17 @@ const InlineStory: FunctionComponent<InlineStoryProps> = (props) => {
       return () => {};
     }
     const element = storyRef.current as HTMLElement;
-    const cleanup = renderStoryToElement(story, element, { autoplay, forceInitialArgs });
+    const cleanup = renderStoryToElement(
+      story,
+      element,
+      {
+        showMain: () => {},
```

Comment:
We could use this to show a spinner or something until the story has rendered but I'm not sure there's much call for that.